app/src/main/java/com/myapp/travelize/authentication/LoginFragment.kt

```diff
@@ -137,5 +156,22 @@ class LoginFragment : Fragment() {
             }
     }
 
+    //textwatcher
+    private val textWatcher  = object : TextWatcher {
```

Comment:
can you use intuitive and specific names like signInTextWatcher for the sake of code maintenance?